server/src/utils/db.js

```diff
@@ -0,0 +1,34 @@
+import 'dotenv/config';
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: String(process.env.DB_PASSWORD),
+  database: process.env.DB_DATABASE,
+  dialect: 'postgres',
+});
+
+export const connectDB = async () => {
+  try {
+    await client.authenticate();
+    console.log('Connection to PostgreSQL has been established successfully.');
+  } catch (error) {
+    console.error('Unable to connect to PostgreSQL:', error);
+    process.exit(1);
+  }
+};
+
+export const syncDB = async () => {
+  try {
+    await client.sync({ force: false });
+    console.log('Database synchronized');
+  } catch (error) {
+    console.error('Failed to synchronize database:', error);
+  }
+};
+
+console.log('DB_HOST:', process.env.DB_HOST);
+console.log('DB_USER:', process.env.DB_USER);
+console.log('DB_PASSWORD:', process.env.DB_PASSWORD);
```

Comment:
Logging the database password is a security risk. Consider removing this log statement to prevent exposing sensitive information.